### PR TITLE
Show images tooltip

### PR DESCRIPTION
**Pull request by loganhz**
Proposed changes
======
Show images tooltip when a pod has multiple images

Types of changes
======
- New feature (non-breaking change which adds functionality)

Linked Issues
======
https://github.com/rancher/rancher/issues/17043
 

Further comments
======

### CODE REVIEW

File: app/components/accordion-pod/template.hbs [hunk 1]
```diff
@@ -32,10 +32,17 @@
...
         </td>
         <td data-title="{{dt.displayImage}}">
-          <small>{{inst.displayImage}}</small>
+          {{#tooltip-element
+              type="tooltip-basic"
+              tooltipTemplate='tooltip-multi-lines'
+              model=inst.displayImages
+              inlineBlock=true
+          }}
+            <small>{{inst.displayImage}}</small>
```
Comment (westlywright) [inline app/components/accordion-pod/template.hbs]:
```suggestion
            <span class="text-small">{{inst.displayImage}}</span>
```

File: app/components/pod-row/template.hbs [hunk 2]
```diff
@@ -26,7 +26,14 @@
...
   </td>
   <td data-title="{{dt.image}}" class="clip top-half">
-     <small>{{model.displayImage}}</small>
+      {{#tooltip-element
+          type="tooltip-basic"
+          tooltipTemplate='tooltip-multi-lines'
+          model=model.displayImages
+          inlineBlock=true
+      }}
+        <small>{{model.displayImage}}</small>
```
Comment (westlywright) [inline app/components/pod-row/template.hbs]:
```suggestion
            <span class="text-small">{{model. displayImage}}</span>
```

File: app/components/workload-row/template.hbs [hunk 3]
```diff
@@ -20,7 +20,14 @@
...
   {{#if showImage}}
     <td data-title="{{dt.image}}" class="clip top-half">
-      <small>{{model.displayImage}}</small>
+      {{#tooltip-element
+          type="tooltip-basic"
+          tooltipTemplate='tooltip-multi-lines'
+          model=model.displayImages
+          inlineBlock=true
+      }}
+        <small>{{model.displayImage}}</small>
```
Comment (westlywright) [inline app/components/workload-row/template.hbs]:
```suggestion
        <span class="text-small">{{model.displayImage}}</span>
```

**Comment by loganhz** [comment]
@westlywright Done. Thanks!

**Pull request merged**